ray-operator/controllers/ray/common/service.go

```diff
@@ -323,15 +323,6 @@ func getServicePorts(cluster rayv1.RayCluster) map[string]int32 {
 		ports = getDefaultPorts()
 	}
 
-	// Check if agent port is defined. If not, check if enable agent service.
```

Comment:
Is the dashboard agent port user-facing?

Reply:
The platform engineer must be aware of it to know what ports to open on their cloud provider for Ray's internal communications to work.  That's why it's documented here: https://docs.ray.io/en/latest/ray-core/configure.html?highlight=ports#all-nodes

But as far as I know the data scientist / ML engineer has no reason to use this port or know about this port.

Reply:
Makes sense to me.